src/main/modules/session/index.ts

```diff
@@ -39,7 +39,7 @@ export class SessionStorage {
       const client = redis.createClient({
         host: redisHost as string,
         password: config.get('session.redis.key') as string,
-        port: 6380,
+        port: 6379,
```

Comment:
Please exclude this file change.

Reply:
Please do not change this port as it corresponds to higher env not local. Suggested to create a new env variable to set this port (Not required as of now can be taken as clean-up process)